coralproject/talk
# [CORL-2478] Upgrade Relay to version 13.0.2

## What does this PR do?

Updates Relay to version 13.0.2 and upgrades all the corresponding underlying graphQL dependencies necessary to enable this.

Also adds new schema combination into a new root `__generated__` folder + build steps to enable this. After a build, if you look in the folder, you'll see it generates new combined schemas that incorporate the server's `schema.graphql` along with the framework's `local.graphql` and also the section of the site's `local.graphql` (i.e. the stream, admin, auth, etc).

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No indexes added.

## How do I test this PR?

- Perform a build
- Run the app locally
- Test anything you feel like to make sure that the site is functioning the same as before this dependency update
 
## How do we deploy this PR?

No special considerations.
